profiler/include/profile_grouped_gemm_impl.hpp

```diff
@@ -83,7 +83,7 @@ void profile_grouped_gemm_impl(int do_verification,
     std::vector<Tensor<BDataType>> b_k_n;
     std::vector<Tensor<CDataType>> c_m_n_device_results;
 
-    for(int i = 0; i < Ms.size(); i++)
+    for(int i = 0; i < ck::type_convert<int>(Ms.size()); i++)
```

Comment:
use group_count defined above.

Reply:
also use const std::vector<int>& for function parameters starting line 53.

Reply:
Addressed by [57f2d3c](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/212/commits/57f2d3c31e163a8525856fb3b1cae2cb09ddbff9)

Reply:
also we already have group_count from line 74, so why not use it to replace Ms.size()?

Reply:
Addressed by [594a4c2](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/212/commits/594a4c22d129e4eac6930355ea180a0e156735e5)